### ISSUE TITLE
dart migrate failure

### ISSUE THREAD

**Issue by jwren** [dart-lang/sdk#40797]
```
~/dart-sdk/dart-sdk_2.8.0-dev.9.0/bin/dart pkg/dartdev/bin/dartdev.dart migrate pkg/dartdev/
Migrating pkg/dartdev/

Analyzing project...           /{"code":"INVALID_FILE_PATH_FORMAT","message":"Invalid file path format: pkg/dartdev/"}
```

At first I thought it was due to a missing / after pkg/dartdev/.. but after adding it I still see the error.

@devoncarew Why the error, what does it actually mean?

Also, we shouldn't show the JSON output like this to the user.

**Comment by devoncarew** [comment]
This may mean that we're feeding a relative path into the analysis server? It only wants absolute paths for many things.

**Comment by jwren** [comment]
Sounds likely.... I remember seeing this error when putting together `analyze`.

**Comment by devoncarew** [comment]
fixed in 15a3141
